mozilla/foundation.mozilla.org
# updated publication and blog pages to work with get_page_authors tag

Closes #7333 


**Link to sample test page**:

Test **Blog** Page with Author Images: https://foundation-s-7333-autho-4onovn.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/
Test **Publication** Page with Author Images: https://foundation-s-7333-autho-4onovn.herokuapp.com/en/publication-page-with-child-article-pages/

**Steps To Test**:

1. Visit the two pages above
2. Ensure that they both are displaying authors with and without images correctly. 
3. If they are, testing is complete! 

## Checklist

**Changes in Models:**
